From 0420d82b6e30679489a4f82c11298b13197c8b98 Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Mon, 2 Jun 2014 18:11:44 -0400
Subject: [PATCH] libarchive: disable lz4

This fixes a build error when the host has lz4 installed locally.
---
 package/libarchive/libarchive.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/libarchive/libarchive.mk b/package/libarchive/libarchive.mk
index ccda18386f..9e84083885 100644
--- a/package/libarchive/libarchive.mk
+++ b/package/libarchive/libarchive.mk
@@ -125,6 +125,7 @@ HOST_LIBARCHIVE_CONF_OPTS = \
 	--without-lzo2 \
 	--without-nettle \
 	--without-openssl \
+	--without-lz4 \
 	--without-lzma
 
 $(eval $(autotools-package))
-- 
2.17.1

